torch/onnx/_internal/exporter.py

```diff
@@ -890,10 +890,10 @@ def pre_export_passes(
     # Run ShapeInferenceWithFakeTensor to get static shape of nodes for op_level_debug purposes
     # The pass added nodes with static shape into original node metadata:
     # node.meta["static_shape"]: FakeTensor/int/float/SymInt/SynFloat
-    if options.op_level_debug:
-        module = passes.ShapeInferenceWithFakeTensor(diagnostic_context, module).run(
-            *fx_module_args
-        )
+    # if options.op_level_debug:
+    #     module = passes.ShapeInferenceWithFakeTensor(diagnostic_context, module).run(
+    #         *fx_module_args
+    #     )
```

Comment:
Delete

Reply:
nit: reminder to delete this + comment + the pass code itself.